SAGE Journals.js

```diff
@@ -109,8 +98,8 @@ function scrape(doc, url) {
 		//and will therefore simply delete the later in cases both
 		//dates are present.
 		//Z.debug(text);
-		if (text.indexOf("DA  - ") > -1) {
-			text = text.replace(/Y1 - .*\r?\n/, '');
+		if (text.includes("DA  - ")) {
+			text = text.replace(/Y1[ ]{2}- .*\r?\n/, '');
```

Comment:
If you want 2 whitespaces, it would be better to use `\s{2}` instead of `[ ]{2}` for better unicode compatibility

Reply:
accepted your regex suggestion